Dockerfile

```diff
@@ -66,40 +77,65 @@ VOLUME ["/data", "/var/log/nginx", "/var/log/wsgi"]
 # Expose ports. (http, connection to DB, websocket port)
 EXPOSE 80 443 8005
 
+##############################################
+# APPLICATION
+
 # APP_HOME
 ENV APP_HOME /home/app/django
-
-# Add project directory to docker
-ADD ./ /home/app/django
-
 WORKDIR /home/app/django
 
-RUN cat docker/version >> /etc/profile
+# Install Ruby dependencies
+COPY Gemfile ./
```

Comment:
🙋‍♂️ There's a `Gemfile.lock` committed in the repository, so to avoid installing different versions in the Docker image, we would need to copy that too before `bundle install`:

```
COPY Gemfile* ./
RUN bundle install
```